src/Components/Admin/Activities/Form/index.jsx

```diff
@@ -61,7 +61,7 @@ const Form = () => {
       ) : (
         <>
           <form className={styles.form} onSubmit={handleSubmit(onSubmit)}>
-            <div className={styles.divContainer}>
+            <div className={styles.divContainer} testId="add-form">
```

Comment:
When you are adding the test id to a non shared component, in this case a simple html tag the property should be `data-testId` instead `testId`. You just use `testId` to add a prop to a shared component

```suggestion
            <div className={styles.divContainer} data-testId="add-form">
```